custom/icds_reports/migrations/0104_citus_composite_key.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.conf import settings
+from corehq.sql_db.operations import HqRunPython
+
+
+def _citus_composite_key_migration(apps, schema_editor):
+    models_to_update = [
+        'aggregateawcinfrastructureforms',
+        'aggregatebirthpreparednesforms',
+        'aggregateccsrecorddeliveryforms',
+        'aggregateccsrecordpostnatalcareforms',
+        'aggregateccsrecordthrforms',
+        'aggregatechildhealthdailyfeedingforms',
+        'aggregatechildhealthpostnatalcareforms',
+        'aggregatechildhealththrforms',
+        'aggregatecomplementaryfeedingforms',
+        'aggregategrowthmonitoringforms',
+        'dailyattendance',
+        'awwincentivereport'
+    ]
+    if not getattr(settings, 'IS_ON_CITUSDB_BACKEND', False):
```

Comment:
This flag doesn't exist - I think you can use the `UNIT_TEST` flag for now

Reply:
We can create one and use that? I will use `UNIT_TEST` for now.